core/pkg/ingress/controller/controller.go

```diff
@@ -620,9 +620,8 @@ func (ic *GenericController) getBackendServers() ([]*ingress.Backend, []*ingress
 			}
 
 			if rule.HTTP == nil &&
-				len(ing.Spec.TLS) == 0 &&
```

Comment:
If I just create an ingress with a TLS section, doesn't that mean I want tls routing?